driving game/main.js

```diff
@@ -21,4 +25,15 @@ document.addEventListener('keydown', function (e) {
     $car.className = 'east';
     carDirection.currentDirection = 'east';
   }
+  if (e.key === ' ') {
+    startCar()
+  }
 });
+
+function startCar () {
+  $car.style.left = offSet[0] + 10 + 'px';
+  offSet[0] = $car.offsetLeft;
+  offSet[1] = $car.offsetTop;
+  carDirection.location = offSet;
+  intervalId = setInterval(startCar, 16)
+}
```

Comment:
This is using recursion, probably not on purpose.  log something in the console in this function to see what im talking about

Reply:
Does that just mean startCar is being called again every time the code on line 38 is run? So setInterval should be taken out of the function and instead get called on the keydown event?

Reply:
yes exactly

Reply:
every interval starts a new interval

Reply:
![startcar](https://user-images.githubusercontent.com/60412330/101970127-72840d00-3bdd-11eb-8fb2-219590e62d82.gif)